### DISCUSSION TITLE
exception/npe

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#8] [Announcements]
# exception/npe

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/exception/npe.html

**Comment by auqaoasis** [comment]
真有意思，脑海中突然就有整张图了！！
